src/spyglass/common/common_nwbfile.py

```diff
@@ -310,26 +311,16 @@ def get_abs_path(analysis_nwb_file_name):
         analysis_nwb_file_abspath : str
             The absolute path for the given file name.
         """
-        try:
-            base_dir = Path(dj.config["stores"]["analysis"]["location"])
-        except KeyError as e:
-            raise KeyError(
-                "You must set the 'analysis' store location in dj.config"
-            ) from e
-
         # see if the file exists and is stored in the base analysis dir
-        test_path = str(base_dir / analysis_nwb_file_name)
+        test_path = str(Path(analysis_dir) / analysis_nwb_file_name)
```

Comment:
What's the motivation behind str->Path->str? To normalize regardless of whether or not there's a `/` at the end? I think settings.py checks that all paths are valid and then passes them as str, just because that was the norm previously. I think we'd be better off migrating to pathlib objs in the future